vertx-mail-client/src/main/java/io/vertx/ext/mail/impl/Utils.java

```diff
@@ -22,13 +22,19 @@
 import java.util.ArrayList;
```

Comment:
I assume the license header is the same as in the other files, are you sure its wrong?